endpoints/getting-started-grpc/README.md

```diff
@@ -0,0 +1,139 @@
+# Google Cloud Endpoints Sample for Go using gRPC
+
+This sample demonstrates how to use Google Cloud Endpoints using Go and gRPC.
+
+## Test the code locally (optional)
+
+Run the backend using `go run`:
+
+```bash
+$ go run server/main.go
+```
+
+Sent a request:
+
+```bash
+$ go run client/main.go
+2017/03/30 17:08:32 Greeting: Hello world
+```
+
+## Deploying service config
+
+1. First, generate `out.pb` from the proto file:
+
+    ```bash
+    protoc --include_imports --include_source_info helloworld/helloworld.proto --descriptor_set_out out.pb
+    ```
+
+1. Edit `api_config.yaml`. Replace `YOUR_PROJECT_ID`:
+
+    ```yaml
+    name: hellogrpc.endpoints.YOUR_PROJECT_ID.cloud.goog
+    ```
+
+1. Deploy your service:
+
+    ```bash
+    gcloud service-management deploy out.pb api_config.yaml
+    ```
+
+    Your config ID should be printed out, it looks like `2017-03-30r0`.
+    Take a note of it, you'll need it later.
+
+    You can list the config IDs using this command:
+
+    ```bash
+    gcloud service-management configs list --service hellogrpc.endpoints.YOUR_PROJECT_ID.cloud.goog
+    ```
+
+## Building the server's Docker container
+
+Build and tag your gRPC server, storing it in your private container registry:
+
+```bash
+gcloud container builds submit --tag gcr.io/YOUR_PROJECT_ID/go-grpc-hello:1.0 .
```

Comment:
you could use `"gcr.io/$(gcloud config get-value project)/go-grpc-hello:1.0"` to avoid having people trying to deploy to `YOUR_PROJECT_ID` and being confused